### PR TITLE
Update Web-Interface.md

### PR DESCRIPTION
**Pull request by Petrprogs**
Add Web Interface Review. That's not all :)

### CODE REVIEW

**Review by ckerr — COMMENTED** [review]
These are great; keep 'em coming! :green_heart:

**Comment by Petrprogs** [comment]
> These are great; keep 'em coming! green_heart

Thank you!

File: docs/Web-Interface.md [hunk 1]
```diff
@@ -34,6 +34,57 @@ Click "Close" button. Done!
...
-If the web browser and the Transmission daemon are on the machine you can use http\://127.0.0.1:9091/
-9091 is the default remote control port specified in [Transmission configuration](Editing-Configuration-Files.md) or in preferences of [Windows](https://github.com/transmission/transmission/edit/main/docs/Web-Interface.md#1-for-windows) or [Linux](https://github.com/transmission/transmission/edit/main/docs/Web-Interface.md#2-for-linux) client.
+If the web browser and the Transmission daemon are on the machine you can use http://127.0.0.1:9091/
+9091 is the default remote control port specified in [Transmission configuration](Editing-Configuration-Files.md) or in preferences of [Windows](https://github.com/transmission/transmission/blob/main/docs/Web-Interface.md#1-for-windows) or [Linux](https://github.com/transmission/transmission/blob/main/docs/Web-Interface.md#2-for-linux) client.
+
+## Web Interface Overview ##
+### 1. Main Screen ###
+![image](https://user-images.githubusercontent.com/76732207/181889508-c926e97c-f51a-4562-bf8b-75e4be672786.png)
+
+| Number On Picture | Discription                 |
```
Comment (RobFog) [inline docs/Web-Interface.md:44]: "Discription" should read "Description" throughout.

Comment (Petrprogs) [reply docs/Web-Interface.md:44]: Fixed by #3557

**Pull request merged**